### PR TITLE
fix #70722 ero-top.info

### LINKED ISSUE #70722: ero-top.info
### Issue URL (Ads)
[NSFW] [https://ero-top.info/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fero-top.info%2F)

### [NSFW] Screenshots
<details>
  <summary>Screenshot 1</summary>

  [Screenshot 1](https://reports-img.adguard.com/lsYqV9s.jpg)
</details>

### System configuration

Information | value
--- | ---
Platform: | uBlock Origin
Browser: | Firefox
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>EasyList<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances

### PR DESCRIPTION
**Pull request by Yuki2718**
https://github.com/AdguardTeam/AdguardFilters/issues/70722
Not sure if should use basic rule than cosmetic.

### CODE REVIEW

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -49,6 +49,7 @@ crazymailing.com,lotzon.pro,novosti-n.org,prokazan.ru,20minut.ua,alnews.com.ua,g
 ! vast ads on anime sites - for example: animego.org, yummyanime.club, etc
 /:\/\/[0-9]?[a-z0-9]{11,}(\.com|\.in\.net)/$xmlhttprequest,domain=aniqit.com|kodik.info|start.u-cdn.top|ustore.bz|u-stream.in
 !
+ero-top.info##.remove_in_mobile
```
Comment (zloyden) [inline RussianFilter/sections/specific.txt:52]: `||ero-top.info/uploads/banner*.jpg`
Also please add `ero-top.info#?##menu > div.shap:contains(Реклама)`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 1 above)
Comment (Yuki2718) [reply RussianFilter/sections/specific.txt:52]: Added.

File: RussianFilter/sections/specific.txt [hunk 2]
```diff
@@ -49,6 +49,8 @@ crazymailing.com,lotzon.pro,novosti-n.org,prokazan.ru,20minut.ua,alnews.com.ua,g
 ! vast ads on anime sites - for example: animego.org, yummyanime.club, etc
 /:\/\/[0-9]?[a-z0-9]{11,}(\.com|\.in\.net)/$xmlhttprequest,domain=aniqit.com|kodik.info|start.u-cdn.top|ustore.bz|u-stream.in
 !
+||ero-top.info/uploads/banner*.jpg
+ero-top.info##.remove_in_mobile
```
Comment (zloyden) [inline RussianFilter/sections/specific.txt:53]: I meant to exchange this `ero-top.info##.remove_in_mobile` rule for blocking the request :)

Comment (Yuki2718) [reply RussianFilter/sections/specific.txt]: Oh, I mistook. Removed.

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
Thanks:)

File: RussianFilter/sections/css_extended.txt [hunk 3]
```diff
@@ -14,6 +14,8 @@ kino-teatr.ru#?##root div[class^="relap-"]:contains(РЕКОМЕНДУЕТ) div[
 underverse.su,underver.se#?##sidebar1-wrap > h3:contains(Реклама)
 underverse.su,underver.se#?##sidebar1-wrap > h3:contains(Реклама) + div.sb_back
 underverse.su,underver.se#?##sidebar1-wrap > h3:contains(Реклама) + div.sb_back + div.sb_bottom
+! https://github.com/AdguardTeam/AdguardFilters/pull/70744
```
Comment (zloyden) [inline RussianFilter/sections/css_extended.txt:17]: Here we use a task link (not pull request one). I will change it.

Comment (Yuki2718) [reply RussianFilter/sections/css_extended.txt:17]: Sorry, was mistake.

**Pull request merged**